.travis.yml

```diff
@@ -47,6 +47,7 @@ before_script:
   - sudo add-apt-repository ppa:ubuntu-toolchain-r/test --yes
   - sudo apt-get update
   - sudo apt-get install google-chrome-beta
+  - sudo apt-get --only-upgrade install google-chrome-stable
```

Comment:
Just out of curiosity, why do we install chrome beta and then upgrade `google-chrome-stable`? Could we skip installing chrome beta, since it appears headless chrome has made it's way into stable?

Reply:
yeah turns out installing chrome beta doesn't help because it wasn't being used anyway. Its left over from when I was trying to figure out how to get an updated chrome installed. I'll remove it.